pkgs/applications/editors/vim/plugins/generated.nix

```
@@ -3346,6 +3346,18 @@ final: prev:
     meta.homepage = "https://github.com/junegunn/gv.vim/";
   };
 
+  hare-vim = buildVimPluginFrom2Nix {
+    pname = "hare.vim";
+    version = "2022-07-02";
+    src = fetchFromSourceHut {
```

Almost there, you've missed this part :)
```suggestion
    src = fetchFromSourcehut {
```
My initial suggestion had the _H_ capitalized, sorry.

---

Ah, no wonder I found no example usages!